src/data/polynomial/denoms_clearable.lean

```diff
@@ -70,3 +70,27 @@ theorem denoms_clearable_nat_degree
 denoms_clearable_of_nat_degree_le f.nat_degree a bu f le_rfl
 
 end denoms_clearable
+
+open ring_hom
+
+/--  Evaluating a polynomial with integer coefficients at a rational number and clearing
+denominators, yields a number greater than or equal to one.  The target can be any
+`linear_ordered_field K`.
+The assumption on `K` could be weakened to `linear_ordered_comm_ring` assuming that the
+image of the denominator is invertible in `K`. -/
+lemma one_le_pow_mul_abs_eval_div {K : Type*} [linear_ordered_field K] {f : polynomial ℤ}
+  {a b : ℤ} (b0 : (0 : K) < b) (fab : eval ((a : K) / b) (f.map (algebra_map ℤ K)) ≠ 0) :
```

Comment:
I think I prefer the other form of `b0`. It's best to keep the assumptions as "simple" as possible.

Reply:
I agree with you.  Once I am back at my computer, I will push the change!